core/src/subgraph/runner.rs

```diff
@@ -1061,11 +1062,14 @@ async fn update_proof_of_indexing(
 
         // Put this onto an entity with the same digest attribute
         // that was expected before when reading.
-        let new_poi_entity = entity! {
-            entity_cache.schema =>
-            id: entity_key.entity_id.to_string(),
-            digest: updated_proof_of_indexing,
-        }?;
+        let data = vec![
```

Comment:
This form seems a bit harder to read and seems a bit longer too. Is this to avoid changing the macro?